libs/environment/src/components/network-switcher/network-switcher.tsx

```diff
@@ -88,10 +84,9 @@ export const NetworkSwitcher = ({
     [setOpen, setAdvancedView]
   );
 
-  const dropdownTriggerClasses = classNames('hover:!bg-neutral-700', {
-    'dark:text-white dark:bg-black text-black bg-white': !fixedBg,
-    'text-black bg-white': fixedBg === 'light',
-    'text-white bg-black': fixedBg === 'dark',
+  const dropdownTriggerClasses = classNames({
+    'text-black hover:!bg-neutral-300': theme === 'light',
```

Comment:
Should be 700 for both themes as the header is black for both. Otherwise it does this on light theme:

![image](https://user-images.githubusercontent.com/1325606/189879764-9636eb65-92dc-4e27-af8f-3ebb5d3a7f3f.png)

Reply:
Not seeing this one

Reply:
Still happening for me:

- Go here https://deploy-preview-1313--vega-trading-app.netlify.app/markets/bb941f84e25b3a06068e33917d01215d56a51e9abff6ff9b9a3f2cf49b495e37
- Switch to light mode
- Connect to wallet
- Hover over dropdown